packages/features/cucumber.js

```diff
@@ -10,6 +10,7 @@ const common = [
   '--require steps/**/*.ts',
   '--require lib/**/*.ts',
   '--format @cucumber/pretty-formatter',
+  '--parallel 3',
```

Comment:
We only have 2 acceptance tests (and 1 is trivial), but it might be useful for the future :)